sdk/ml/azure-ai-ml/azure/ai/ml/entities/_job/job_service.py

```diff
@@ -35,25 +36,41 @@ def __init__(
         *,
         endpoint: Optional[str] = None,
         job_service_type: Optional[Literal["JupyterLab", "SSH", "TensorBoard", "VSCode"]] = None,
```

Comment:
In a separate PR if we can do snake case to pascal case that would be great as well. Because this seems odd one out here.

Reply:
sure